exe/kubernetes-deploy

```diff
@@ -15,18 +15,23 @@ require 'kubernetes-deploy'
 require 'optparse'
 
 skip_wait = false
+template_dir = nil
 ARGV.options do |opts|
   opts.on("-w", "--skip-wait") { skip_wait = true }
+  opts.on("-tDIR", "--template-dir=DIR") { |v| template_dir = v }
   opts.parse!
 end
 
+if !template_dir || template_dir.empty?
+  raise ArgumentError, "--template-dir flag is required"
+end
```

Comment:
Agreed, this should not be a required arg and the default should remain on this side.